tickets/k8s/.dir-locals.el

```diff
@@ -1,17 +1,111 @@
 ((org-mode .
-      ((eval .
-             (defun apisnoop/insert-mock-template ()
-               "Inserts contents of current directory's mock-template.org file into current buffer."
-               (interactive)
-               (let ((mock-template "./mock-template.org"))
-                 (if (file-exists-p mock-template)
-                     (progn (insert-file-contents mock-template)
-                            (normal-mode))
-                   (message "No file named %s found in current directory" mock-template)))))
-       (eval .
-             (defun apisnoop/delete-live-events ()
-               "deletes all live events from audit_events table using psql.  Requires you to have already
+           (
+            (eval . (setq-local RESOURCENAME "RESOURCENAME"))
+            (eval .
+                  (defun apisnoop/insert-mock-template ()
+                    "Inserts contents of current directory's mock-template.org file into current buffer."
+                    (interactive)
+                    (let ((mock-template "./mock-template.org"))
```

Comment:
Might be good to detect IF we are in a org file and if not offer to create one in this folder.